xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -198,9 +215,17 @@ The `CurrentDepth` property tracks the recursive depth of the nested objects / a
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Asynchronously commits any leftover JSON text that has not yet been flushed and releases all resources used by the current instance.</summary>
         <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <remarks>
```

Comment:
Same comment as for Dispose.

Reply:
@ahsonkhan Can you also please confirm these remarks describe the correct behavior?

@rpetrusha Same case, the triple slash comments were properly ported:

https://github.com/dotnet/corefx/blob/2fc59abd09c990121526360d90810e19723cb6cb/src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs#L343-L354

Reply:
Yes, they are valid. If the question is, should the implementation/design change, that's a different story all together :)